testsuite/python/CMakeLists.txt

```diff
@@ -59,8 +63,18 @@ function(PYTHON_TEST)
   set(python_tests ${python_tests} ${TEST_FILE} PARENT_SCOPE)
 endfunction(PYTHON_TEST)
 
-python_test(FILE save_checkpoint.py MAX_NUM_PROC 4)
-python_test(FILE test_checkpoint.py MAX_NUM_PROC 4 DEPENDS save_checkpoint)
+foreach(TEST_COMBINATION lb.cpu-p3m.cpu-lj;lb.gpu-p3m.cpu-lj)
```

Comment:
Could you please add a comment how to add new tests?

Reply:
Done.